control/control_test.go

```diff
@@ -1354,6 +1371,7 @@ func TestProcessMetrics(t *testing.T) {
 
 		// Load plugin
 		_, err := load(c, path.Join(SnapPath, "plugin", "snap-processor-passthru"))
+		So(err, ShouldBeNil)
 		<-lpe.done
 		So(err, ShouldBeNil)
```

Comment:
If the err is going to be checked before the channel returns, then do not need to check the error after the channel returns as well.